packages/qdrant-loader/src/qdrant_loader/core/worker/handlers.py

```
@@ -0,0 +1,93 @@
+        """Execute a job of the given type with the provided payload."""
+        ...
+
+
+class BaseJobHandler(ABC):
+    """Base class for job handlers."""
+
+    async def __call__(self, job_type: str, payload: dict[str, Any]) -> None:
+        """Dispatch to handler method based on job_type."""
+        if job_type == "BULK_INGEST":
+            return await self.handle_bulk_ingest(payload)
+        elif job_type == "INCREMENTAL_PULL":
```

Ruff (0.15.13)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)